src/test/mir-opt/const_prop/aggregate/rustc.main.ConstProp.diff

```diff
@@ -7,7 +7,8 @@
       let mut _2: i32;                     // in scope 0 at $DIR/aggregate.rs:5:13: 5:24
       let mut _3: (i32, i32, i32);         // in scope 0 at $DIR/aggregate.rs:5:13: 5:22
       scope 1 {
-          debug x => _1;                   // in scope 1 at $DIR/aggregate.rs:5:9: 5:10
+-         debug x => _1;                   // in scope 1 at $DIR/aggregate.rs:5:9: 5:10
++         debug x => const 1_i32;          // in scope 1 at $DIR/aggregate.rs:5:9: 5:10
```

Comment:
Shouldn't this transformation result in `_1`'s store being dead? Or would that be in a different pass? Should we be tracking that as well, not just `ConstProp`?

Reply:
We have a `SimplifyLocal` pass that runs right after `ConstProp` which will remove the `_1` local.